modules/avcodec/avcodec.c

```diff
@@ -33,6 +33,7 @@
  *
  \verbatim
       avcodec_h264enc  <NAME>  ; e.g. nvenc_h264, h264_videotoolbox
+      avcodec_h264dec  <NAME>  ; e.g. nvenc_h264, h264_videotoolbox
```

Comment:
perhaps update comment to list a couple of H.264 decoders?

ffmpeg ./configure --list-decoders give me this:

```
h264_crystalhd
h264_cuvid
h264_mediacodec
h264_mmal
h264_qsv
h264_vda
h264_vdpau
```